logaggregator/main.go

```diff
@@ -178,7 +175,7 @@ func (a *Aggregator) readLogsFromConn(conn net.Conn) {
 	}()
 
 	s := bufio.NewScanner(conn)
-	s.Split(rfc6587Split)
+	s.Split(rfc6587.Split)
```

Comment:
Well, I left that in because I thought you wanted to use it (maybe for tests or something). But can easily just nuke it.